05.ls/ls.rb

```diff
@@ -2,16 +2,33 @@
 
 # frozen_string_literal: true
 
+require 'optparse'
+
 COLUMN_COUNT = 3
 
 def main
+  is_option_all = false
+
+  opt = OptionParser.new
+  opt.on('-a') do |has_value|
+    is_option_all = true if has_value
+  end
```

Comment:
ブロック内のコードが実行されるのはオプションが指定されていたときだけなので、以下のコードでも同じ動きになるはずです。

```suggestion
  opt.on('-a') { is_option_all = true }
```

Reply:
修正しました！